src/main/java/com/puppycrawl/tools/checkstyle/utils/JavadocUtil.java

```diff
@@ -408,4 +408,46 @@ else if (sibling.getType() == TokenTypes.SINGLE_LINE_COMMENT) {
                 || BlockCommentPosition.isOnPackage(blockComment));
     }
 
+    /**
+     * Gets the content of inline custom tag.
+     *
+     * @param javadocInlineTag javadoc inline tag node.
+     * @return String consisting of the content of inline custom tag.
+     */
+    public static String getContentOfInlineCustomTag(DetailNode javadocInlineTag) {
+        final DetailNode[] childrenOfJavadocInlineTag = javadocInlineTag.getChildren();
+        DetailNode nodeToBeProcessed = childrenOfJavadocInlineTag[2];
+        final DetailNode endingNode =
+                childrenOfJavadocInlineTag[childrenOfJavadocInlineTag.length - 1];
+        final StringBuilder customTagContent = new StringBuilder(400);
+
+        while (nodeToBeProcessed != endingNode) {
+            customTagContent.append(extractContentInlineCustomTag(nodeToBeProcessed));
+            nodeToBeProcessed = getNextSibling(nodeToBeProcessed);
+        }
+        return customTagContent.toString();
+    }
+
+    /**
+     * Extracts the content of inline custom tag recursively.
+     *
+     * @param node DetailNode
+     * @return Text of the node supplied, concatenated text if that node has children.
+     */
+    private static String extractContentInlineCustomTag(DetailNode node) {
+        final StringBuilder result = new StringBuilder(160);
+        final DetailNode[] children = node.getChildren();
+        if (children.length == 0) {
```

Comment:
`node.hasChildren()`

Reply:
It's not `DetailAst` but can create such a method for `DetailNode` if you want.

Reply:
sorry, I didnt see it is different type